### ISSUE TITLE
Frozen in windowed mode when exiting the program

### ISSUE THREAD

**Issue by ShonRT** [coelckers/prboom-plus#16]
Whenever I exit this version, the game minimizes itself into a windowed mode and stay frozen like that for up to 8 or 10 seconds sometimes.

**Comment by fabiangreffrath** [comment]
Does it make a difference if you disable "Show ENDOOM screen" in the General settings?

**Comment by ShonRT** [comment]
No, it doesn't

**Comment by fabiangreffrath** [comment]
Does it play the quit sound?

**Comment by ShonRT** [comment]
No, the game just goes into a windowed mode on whatever screen I quit the game on and freezes like that for a few seconds before closing. Is there anyway to send you a screenshot?

**Comment by fabiangreffrath** [comment]
Does it play the quit sound?

**Comment by ShonRT** [comment]
It doesn't play the quit sound

**Comment by fabiangreffrath** [comment]
Could you try to replace the bundled SDL DLLs with the ones from the Chocolate Doom 3.0 release? The SDL version that prboom-plus is built against is known buggy in various ways.

**Comment by ShonRT** [comment]
That didn't seem to fix the issue.

**Comment by fabiangreffrath** [comment]
@ShonRT Do you have any chance to check the current sources to see if your problem from 1.5 years ago still persists?

**Comment by MoskovchenkoD** [comment]
I doubt he will come back. Personally, I've never faced such an issue. Windows 7 x64, 1920x1080 monitor
